### PR TITLE
Website/add metadata

### PR DESCRIPTION
**Pull request by xuechunL**
add some meta-data for auto deploy website.

### CODE REVIEW

File: op-guide/mysql-compatibility.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+title: Compatibility with MySQL
+category: Compatibility and comparison with MySQL
```
Comment (andelf) [inline op-guide/mysql-compatibility.md]: s/comparison/Comparison/

File: op-guide/pd-control.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+---
+title: PD Control instructions
```
Comment (andelf) [inline op-guide/pd-control.md]: Instructions

File: tools/loader.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+---
+title: Loader instructions
```
Comment (andelf) [inline tools/loader.md]: Instructions

File: op-guide/history-read.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+---
+title: Reading data from history versions
```
Comment (andelf) [inline op-guide/history-read.md]: title case, capitalize every first letter

**Comment by andelf** [comment]
LGTM

File: README.md [hunk 5]
```diff
@@ -1,37 +1,33 @@
...
 + Installation and Deployment
-	+ [Deployment Recommendation](op-guide/recommendation.md)
-	+ [Downloading the Binary](op-guide/binary-deployment.md#download-and-decompress-the-official-binary-package)
-	+ [Binary Deployment (Recommended)](op-guide/binary-deployment.md)
-	+ [Docker Deployment](op-guide/docker-deployment.md)
-	+ [Multi-site Deployment](op-guide/location-awareness.md)
-	+ [Configuration and options](op-guide/configuration.md)
+  - [Deployment Recommendation](op-guide/recommendation.md)
+  - [Downloading the Binary](op-guide/binary-deployment.md#download-and-decompress-the-official-binary-package)
+  - [Binary Deployment (Recommended)](op-guide/binary-deployment.md)
```
Comment (GregoryIan) [inline README.md]: s/Recommended/recommended

File: op-guide/monitor-overview.md [hunk 6]
```diff
@@ -14,8 +19,8 @@ Prometheus consists of multiple components. Currently, TiDB uses the following o
 
 The diagram is as follows:
 
-<img src="prometheus-in-TiDB.png" width="500">
+<img src="images/prometheus-in-TiDB.png" width="500">
```
Comment (GregoryIan) [inline op-guide/monitor-overview.md]: ../media/prometheus-in-tidb.png

File: op-guide/monitor-overview.md [hunk 7]
```diff
@@ -14,8 +19,8 @@ Prometheus consists of multiple components. Currently, TiDB uses the following o
...
 
 ## About Grafana in TiDB
 Grafana is an open source project for analysing and visualizing metrics. TiDB uses Grafana to display the performance metrics as follows:
-![screenshot](grafana-screenshot.png)
+![screenshot](media/grafana-screenshot.png)
```
Comment (GregoryIan) [inline op-guide/monitor-overview.md]: ../media, ditto

File: op-guide/monitoring-tidb.md [hunk 8]
```diff
@@ -128,7 +133,7 @@ You can get the following metrics for each component:
 
 See the following diagram for the deployment architecture:
 
-![image alt text](monitoring-tidb.png)
+![image alt text](media/monitoring-tidb.png)
```
Comment (GregoryIan) [inline op-guide/monitoring-tidb.md]: ditto

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**